src/jit/codegenarmarch.cpp

```diff
@@ -820,7 +820,7 @@ void CodeGen::genPutArgStk(GenTreePutArgStk* treeNode)
                 {
                     // Load from our varNumImp source
                     emit->emitIns_R_R_S_S(INS_ldp, emitTypeSize(type0), emitTypeSize(type1), loReg, hiReg, varNumInp,
-                                          0);
+                                          structOffset);
```

Comment:
Do we need to update the comment in 811 above?

What happens when structOffset exceeds range of ldp?

Reply:
We already have that issue in the existing code for the non-var case. Is it handled elsewhere?

Reply:
looks like it has to be `<= MAX_PASS_MULTIREG_BYTES` which is 32 (max size of HFA)

Reply:
Looks like it is handled in emitIns_R_R_S_S by using a register